econml/cate_estimator.py

```diff
@@ -685,6 +776,51 @@ def summary(self, alpha=0.1, value=0, decimals=3, feature_names=None, treatment_
         if len(smry.tables) > 0:
             return smry
 
+    def shap_values(self, X, *, feature_names=None, treatment_names=None, output_names=None):
+        d_t = self._d_t[0] if self._d_t else 1
```

Comment:
maybe here build a _shap_explain_linear_model_cate in shap.py that contains this logic here and also re-uses the utilities for creating the treatment names in the beginning and the utility that transposes the shap values at the end.